draft-ietf-ppm-dap.md

```diff
@@ -1288,10 +1289,21 @@ report for this reason, it SHOULD abort the upload interaction and alert the
 Client with error `reportTooEarly`. In this situation, the Client MAY re-upload
 the report later on.
 
-If the Leader's input share contains an unrecognized report extension, or if
-two report extensions have the same ExtensionType, then the Leader MAY abort
-the upload request with error "invalidMessage". Note that this behavior is not
-mandatory because it requires the Leader to decrypt its input share.
+If the Leader's input share contains an unrecognized report extension, then the
+Leader MAY abort the upload request with error "unsupportedExtension." If the
+Leader does abort for this reason, it MAY indicate the unsupported extensions
+in the resulting problem document via an extension field
+"unsupported_extensions" on the problem document; this field MUST contain an
+array of numeric values indicating the extension code points which were not
```

Comment:
I think we need to spell out "array of numeric values" in more detail. Is this just a vector of bytes, where each byte is a different error variant? Should this be base64url encoded?

Reply:
A problem details document is a JSON object; with that context, I think this meaning is clear -- an example value might be `[23, 45, 52]`.

I added a reference to the relevant section of the problem details RFC 9457 to make this clear.

Reply:
Great point. In that case, my only suggestion would be to add the example to the text.